src/peft/peft_model.py

```diff
@@ -950,7 +951,30 @@ def set_additional_trainable_modules(self, peft_config, adapter_name):
             else:
                 self.modules_to_save.update(peft_config.modules_to_save)
             # this may add a new ModulesToSaveWrapper
-            _set_trainable(self, adapter_name, modules_to_save=peft_config.modules_to_save)
+            _set_trainable(self, adapter_name, module_names=peft_config.modules_to_save, mode="retrain")
+
+        if getattr(peft_config, "trainable_token_indices", None) is not None:
```

Comment:
```suggestion
        if getattr(peft_config, "trainable_token_indices", None):
```

The problem is that `peft_config.trainable_token_indices` is `[]` by default, so we end up in this branch inadvertently. Fixing this will hopefully resolve the failing CI caused by `LNTuning`.